admin_guide/sysctls.adoc

```diff
@@ -0,0 +1,168 @@
+[[admin-guide-sysctls]]
+= Sysctls
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Certain sysctl settings are exposed via Kubernetes, allowing users to modify
+kernel parameters at runtime for namespaces within a container. Only sysctls
```

Comment:
Sysctl settings are exposed via Kubernetes, allowing users to modify certain kernel parameters at runtime for namespaces within a container. Only sysctls that are namespaced can be set independently on pods; if a sysctl is not namespaced (called _node-level_), it cannot be set within {product-title}. Moreover, only those sysctls considered safe are whitelisted by default, other unsafe sysctls can be manually enabled on the node to be available to the user.